rtcp.c

```diff
@@ -1044,3 +1044,345 @@ int janus_rtcp_nacks(char *packet, int len, GSList *nacks) {
 	rtcp->length = htons(words);
 	return words*4+4;
 }
+
+
+inline void set1(guint8 *data,size_t i,guint8 val)
```

Comment:
Just a nit (as this is code specific to rtcp.c and so is never visible outside of here) but please prefix the internal methods as well (e.g., `janus_rtcp_set1` or whatever prefix makes more sense), as it makes for better readability and generated documentation in general. As a side note, please add a space after the commas in signatures as well.